proxy/http/Http1ClientSession.cc

```diff
@@ -82,11 +82,21 @@ Http1ClientSession::Http1ClientSession()
 void
 Http1ClientSession::destroy()
 {
+}
+
+void
+Http1ClientSession::really_destroy()
```

Comment:
Really? Can you please elaborate on why another layer of destroy is needed? It seems fairly ugly to me :-/